src/actions/content.js

```diff
@@ -24,19 +24,21 @@ function* loadContent(action) {
     (action.payload.options && action.payload.options.contentTypes) || [];
   const status =
     (action.payload.options && action.payload.options.status) || null;
+  const sort = (action.payload.options && action.payload.options.sort) || null;
+
   try {
     yield put(resetLoading());
     yield put(showLoading());
 
     const queryString = {
-      sort: {
-        'sort-changed': {
-          path: 'changed',
-          direction: 'DESC',
-        },
-      },
       filter: {},
     };
+
+    if (sort) {
+      const { name, path, direction } = sort;
+      queryString.sort = { [`sort-${name}`]: { path, direction } };
```

Comment:
This is Drupal-specific sorting syntax. For simple sorts like these, I'd recommend sticking to what the spec provides: http://jsonapi.org/format/#fetching-sorting

Reply:
This is never going to be used outside of connecting to Drupal.

Reply:
Well, changing to ```queryString.sort = [`${direction === 'DESC' ? '-' : ''}${direction}`]``` should be easy?